sdk/turing/router/config/router_config.py

```diff
@@ -136,6 +142,13 @@ def default_route_id(self) -> str:
         details="Please use the ensembler properties to configure the final / fallback route.")
     def default_route_id(self, default_route_id: str):
         self._default_route_id = default_route_id
+        # User may directly modify the default_route_id property while it is deprecated.
+        # So, copy to the nop / standard ensembler if set.
+        if hasattr(self, "ensembler"):
```

Comment:
Just curious, wouldn't `if self.ensembler` work? 🤔

Reply:
Not on `__init__` which is why I added this. During init, the default_route_id attribute is created first (followed by the `ensembler` which always reads the value from the `default_route_id` if its own value is not an Ensembler instance.